best.html

```diff
@@ -33,34 +33,34 @@
 an interface to a page of your AUT. The tests then use the methods of
 this page object class whenever they need to interact with that page
 of the UI. The benefit is that if the UI changes for the page, the
-tests themselves don’t need to change, only the code within the page
-object needs to change. Subsequently all changes to support that new
+tests themselves don’t need to change; only the code within the page
```

Comment:
Keep comma. A semicolon should be used when listing multiple statements that each contain a comma, and is superfluous in this context.